publish-package/action.yml

```diff
@@ -43,7 +43,7 @@ runs:
 
       - name: Publish distribution package to PyPI (production)
         uses: pypa/gh-action-pypi-publish@release/v1
-        if: ${{ !env.ACT || inputs.debug_mode == "false" }}
+        if: ${{ !env.ACT || inputs.debug_mode == false }}
```

Comment:
```suggestion
        if: inputs.debug_mode == 'false' && env.ACT != 'true'
```

- ❌  Error in if-expression: "if: inputs.debug_mode == "true"" (Failed to parse: got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter)
- I had to add the single quotes to make it run correctly, actions doesn't support bools as input.
- Had to adjust the `env.ACT` value.
- Step should now run if debug mode is not true and env is not act.